src/error.rs

```diff
@@ -100,5 +100,21 @@ quick_error! {
          InvalidWireMsgFlag {
              display("Could not deserialise a wire message: unexpected message type flag")
          }
+         /// Write error
```

Comment:
I will do a follow-up PR to remove quick_error and we will use err-derive as quinn does. That should give us composable errors based on std::Error